packages/ozone-helper/ozone-type/ozone/model/FlowrWarningParsable.ts

```diff
@@ -0,0 +1,6 @@
+import {UUID} from "./Item";
+
+export class FlowrWarningParsable {
```

Comment:
What is this type ? it's what is parsed by papaparse ?

Reply:
Yes, it's a common parent of all types that will be handled by PapaParse and CsvManager